### PR TITLE
Created draft for React-Firebase Blog

### PR DESCRIPTION
**Pull request by desoga10**

### CODE REVIEW

**Comment by bartosz-io** [comment]
- Avatar image has a doubled extension `.png.png`
- Should be placed in `./themes/my-theme/source/img/contributors/__HERE__`

**Review by bartosz-io — CHANGES_REQUESTED** [review]
Small fixes are needed. Then you can use the link in the PR to see the deployment preview :)

File: _config.yml [hunk 1]
```diff
@@ -402,3 +402,18 @@ contributors:
       twitter: https://twitter.com/isayjilh
       github: https://github.com/jilh
       linkedin: https://www.linkedin.com/in/jilh
+"Deji Adesoga":
```
Comment (bartosz-io) [inline _config.yml:405]: The YAML format is using tabs to create **a list item**. This is not valid. Please look at the examples above.

**Review by bartosz-io — CHANGES_REQUESTED** [review]

File: source/_posts/react-firebase.md [hunk 2]
```diff
@@ -0,0 +1,954 @@
...
+- [How to Integrate Firebase in React](#how-to-integrate-firebase-in-react)
+- [Integrate Bootstrap 5 into React](#integrate-bootstrap-5-into-react)
+- [Designing the User Interface](#designing-the-user-interface)
+- [Adding Data to the Firestore in React](#adding-data-to-the-firestore-in-react)
+- [Fetching Data from the Firestore in React](#fetching-data-from-the-firestore-in-react)
+- [Deleting Data from the Firestore in React](#deleting-data-from-the-firestore-in-react)
+- [Updating Data in the Firestore in React](#updating-data-in-the-firestore-in-react)
+- [How to Integrate the Checkbox Functionality](#how-to-integrate-the-checkbox-functionality)
+- [How to order Data by Timestamp in Firebase](#how-to-order-data-by-timestamp-in-firebase)
+- [Conclusion](#conclusion)
```
Comment (bartosz-io) [inline source/_posts/react-firebase.md:35]: Please use `<!-- toc -->` instead of the contents. Examples are in the other blog posts.

**Review by bartosz-io — CHANGES_REQUESTED** [review]

File: source/_posts/react-firebase.md [hunk 3]
```diff
@@ -0,0 +1,891 @@
+---
+title: Build a To-Do Application with React and Firebase
+contributor: Deji Adesoga
+avatar: deji-adesoga.png
+description: Learn how to use build a CRUD application by using React and Firebase 9.
```
Comment (bartosz-io) [inline source/_posts/react-firebase.md:5]: CRUD or To-Do? :) Let's be consistent, please.

Comment (bartosz-io) [reply source/_posts/react-firebase.md:5]: @desoga10

File: source/_posts/react-firebase.md [hunk 4]
```diff
@@ -0,0 +1,880 @@
...
+
+{% img "dashboard-landing.png" "Firebase dashboard" "lazy" %}
+
+We have now completed the creation of a new Firebase project.
+
+## Creating the Firestore Database
+
+Inside the firebase dashboard, on the left-hand panel, we take the following steps:
+
+• Click on the **Build** dropdown.
```
Comment (bartosz-io) [inline source/_posts/react-firebase.md:62]: Please use `-` for the lists (as in other examples) to have proper semantics.

**Review by bartosz-io — CHANGES_REQUESTED** [review]

File: source/_posts/react-firebase.md (hunk 3 above)
Comment (desoga10) [reply source/_posts/react-firebase.md:5]: I'll change the CRUD to To-Do.

**Review by bartosz-io — APPROVED** [review]

**Pull request merged**